news/2020-05-08-mappers-report-april.md

```diff
@@ -0,0 +1,193 @@
+
+### The osu! Map Cast
+
+Mentioned in [last month's Mappers' Report](https://osu.ppy.sh/home/news/2020-04-11-mappers-report-march), the **osu! Map Cast** is still going strong, with five new episodes! Usually once per week, this podcast hosted by [Cinnabun](https://osu.ppy.sh/users/11868659) features him talking with a wide variety of mapping community members about osu! and beyond. In the past month and a bit, the osu! Map Cast has published new episodes with [the Elite Nominator Cheri](https://www.youtube.com/watch?v=KpuEXzbmrys), [the osu!taiko Nominator incandescence](https://www.youtube.com/watch?v=poaLJGGeD68), [osu! NAT and former video mapping tutorial extraordinaire pishifat](https://www.youtube.com/watch?v=pJwSShQT3eA), [the new osu! Beatmap Nominator wafer](https://www.youtube.com/watch?v=91oq-hgHAdY), and [a group episode with Nevo, Cheri, incandsecence and Cinnabun all together!](https://www.youtube.com/watch?v=J8Wc8yX4ems) The podcast episodes cover a wide variety of topics, both within osu! such as mapping and modding experiences, and outside of osu! such as birds. Be sure to check out more details and links on the [osu! Map Cast forum post](https://osu.ppy.sh/community/forums/topics/1043221) if this so far has spiked your interest!
+
+### Electoz Advanced Modding Guide
+
+Making waves in the modding community recently is the freshly published modding guide by [Electoz](https://osu.ppy.sh/users/6485263) who is a former Beatmap Nominator and long-time mentor in the Mapping Mentorship Program. The modding guide which can be found on their userpage goes in-depth on basically everything you would need to know about modding at a more advanced level. Electoz has fully infused the guide with his own knowledge and experiences, and the guide itself is largely based off how they would mentor successful aspiring modders! If you're into modding, or want to get into it, checking out the Electoz Advanced Modding Guide should definitely be worth your while.
+
+### Super long marathons are now rankable!
+
+Yes, you heard that right. Until recently, extremely long maps could not be ranked due to breaking the score limit, since the scoreboards wouldn't reflect everyone's plays properly anyways. Now the developers have changed it so that super long maps which break the score limit even without mods will have score v2 force enabled. In such cases, these score v2 scores will be submittable, finally allowing super long maps to have proper leaderboards. This means we'll be seeing extremely long maps ranked in the future, which is a really exciting improvement!
```

Comment:
if any are ranked now, an example would be cool

Reply:
Not yet, the ZUN marathon by Yukiyo was disqualified for storyboard stuff and Nevo's famous Oral Cigarettes Compilation is waiting to be his 100th ranked map. Not sure how appropriate it'd be to mention maps that aren't ranked (yet) here.

Reply:
Since there are no ranked maps where this apply to i would leave it out

Reply:
Main reason it's here is it's similar to an RC change (but not literally RC) where restrictions are lifted that's good for people to know about new possibilities